src/dc_key.rs

```diff
@@ -144,13 +156,16 @@ pub unsafe fn dc_key_set_from_base64(
         type_0,
     );
     mmap_string_unref(result);
-    return 1i32;
+
+    true
 }
+
+// TODO is this function used at all? remove? /rtn
```

Comment:
cc @r10s

Reply:
it is used only in the cmdline util from poke_public_key()
however, i used that for testing in the past wrt netpgp, i do not think it is worth maintaining this, so i would suggest to remove poke_public_key() as well as dc_key_set_from_base64() and other old test-functions that are used only from the cmdline tool.

Reply:
Ok. Added an issue for that.

Reply:
+1